app/graphql/queries/popular_product.rb

```diff
@@ -0,0 +1,12 @@
+module Queries
+  class PopularProduct < BaseQuery
+    type Types::ProductType, null: false
+    description "Query popular Product"
+
+    def resolve()
```

Comment:
NB: Parens are optional. General rule of thumb on ruby function calls and declarations is that parens aren't necessary unless you're gonna fill them with something. Empty parens are better to just leave out entirely